doc/ug/conf.py

```diff
@@ -46,10 +46,9 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 
-extensions = ['sphinx.ext.pngmath',
-              'matplotlib.sphinxext.only_directives',
-              'matplotlib.sphinxext.plot_directive',
+extensions = ['sphinx.ext.mathjax',
               'sphinx.ext.autodoc',
+              'sphinx.ext.napoleon',
```

Comment:
As of Sphinx 1.3, the napoleon extension will come packaged with Sphinx under `sphinx.ext.napoleon`. The `sphinxcontrib.napoleon` extension will continue to work with Sphinx <= 1.2.

On my desktop (`pckr192`) I have `Sphinx (sphinx-build) 1.2b3`. What about you?